db/migrate/20230503123427_create_users.rb

```diff
@@ -0,0 +1,14 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :user do |t|
+      t.text :name
+      t.text :photo
+      t.text :bio
+      t.date :createdat
+      t.date :updatedat
```

Comment:
- [optional] The createdat and updatedat columns are automatically created by rails  using the `t.timestamps`. Kindly remove these, please.